### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by FarukhS52**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ To build a specific release, use the command `release --platform="win64"` for ex
 
 ### Running with debug | Inspector
 
-To be able to open Inspector, you will need SDK flavours of NW.js
```
Comment (MrD-RC) [inline README.md:124]: This is not a spelling mistake. Flavour is spelt correctly.

Comment (FarukhS52) [reply README.md:124]: Okkk, i will change it

**Comment by FarukhS52** [comment]
Hey @MrD-RC ,i have made the changes, is it ok now?

**Comment by MrD-RC** [comment]
Thank you 👍🏻

**Pull request merged**